tests/integration/v4/conftest.py

```diff
@@ -0,0 +1,23 @@
+from typing import List
+
+import pytest
+
+
+def pytest_addoption(parser: pytest.Parser) -> None:
+    parser.addoption(
+        "--no-upgrade", action="store_true", default=False, help="do not run upgrade tests"
+    )
+
+
+def pytest_configure(config: pytest.Config) -> None:
+    config.addinivalue_line("markers", "slow: mark test as slow to run")
+
+
```

Comment:
Do we need the `slow` mark? We do not seem to be using it anywhere.

Reply:
oops, this was copied from the example linked in the comment below, but it wasn't working so I ended up defining the marker in the `pyproject.toml` instead.

You're right, this should be removed.

Reply:
https://github.com/canonical/tls-certificates-interface/pull/304/commits/2b6073fd3037015dd32ef49d60a71497a71cf42f